pkg/util/paramtable/rbac_param.go

```diff
@@ -105,10 +205,67 @@ func (p *rbacConfig) init(base *BaseTable) {
 
 	p.CollectionAdminPrivileges = ParamItem{
 		Key:          "common.security.rbac.collection.admin.privileges",
-		DefaultValue: strings.Join(util.CollectionAdminPrivilegeGroup, ","),
+		DefaultValue: strings.Join(collectionAdminPrivilegeGroup, ","),
 		Version:      "2.4.16",
 		Doc:          "Collection level admin privileges",
 		Export:       true,
 	}
 	p.CollectionAdminPrivileges.Init(base.mgr)
 }
+
+func (p *rbacConfig) GetDefaultPrivilegeGroups() []*milvuspb.PrivilegeGroupInfo {
+	privilegeGroupConfigs := []struct {
+		GroupName  string
+		Privileges func() []string
+	}{
+		{"ClusterReadOnly", p.ClusterReadOnlyPrivileges.GetAsStrings},
+		{"ClusterReadWrite", p.ClusterReadWritePrivileges.GetAsStrings},
+		{"ClusterAdmin", p.ClusterAdminPrivileges.GetAsStrings},
+		{"DatabaseReadOnly", p.DBReadOnlyPrivileges.GetAsStrings},
+		{"DatabaseReadWrite", p.DBReadWritePrivileges.GetAsStrings},
+		{"DatabaseAdmin", p.DBAdminPrivileges.GetAsStrings},
+		{"CollectionReadOnly", p.CollectionReadOnlyPrivileges.GetAsStrings},
+		{"CollectionReadWrite", p.CollectionReadWritePrivileges.GetAsStrings},
+		{"CollectionAdmin", p.CollectionAdminPrivileges.GetAsStrings},
+	}
+
+	builtinGroups := make([]*milvuspb.PrivilegeGroupInfo, 0, len(privilegeGroupConfigs))
+	for _, config := range privilegeGroupConfigs {
+		privileges := lo.Map(config.Privileges(), func(name string, _ int) *milvuspb.PrivilegeEntity {
+			return &milvuspb.PrivilegeEntity{Name: name}
+		})
+		builtinGroups = append(builtinGroups, &milvuspb.PrivilegeGroupInfo{
+			GroupName:  config.GroupName,
+			Privileges: privileges,
+		})
+	}
+	return builtinGroups
+}
+
+func (p *rbacConfig) GetDefaultPrivilegeGroup(privName string) *milvuspb.PrivilegeGroupInfo {
+	for _, group := range p.GetDefaultPrivilegeGroups() {
+		if group.GroupName == privName {
+			return group
+		}
+	}
+	return nil
+}
+
+func (p *rbacConfig) GetDefaultPrivilegeGroupNames() []string {
+	return lo.Keys(builtinPrivilegeGroups)
+}
+
+func (p *rbacConfig) IsCollectionPrivilegeGroup(privName string) bool {
```

Comment:
nit: it is unusual to write implementation details in configuration declarations.